<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfirmTransferDialog</class>
 <widget class="QDialog" name="ConfirmTransferDialog">
  <property name="windowTitle">
   <string>Confirm Transfer</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelName">
     <property name="text">
      <string>Target Playlist Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="playlistNameEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelDesc">
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="playlistDescEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelOptions">
     <property name="text">
      <string>Options</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="skipExistingCheckbox">
       <property name="text">
        <string>Skip tracks already in target</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="buttonsLayout">
     <item>
      <spacer name="hspacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="confirmButton">
       <property name="text">
        <string>Start Transfer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 </ui>


